### PR TITLE
Refactor

### LINKED ISSUE #35: Load quotes and tags in advance

### PR DESCRIPTION
**Pull request by athnme**

### CODE REVIEW

**Review by mhstrkmp — APPROVED** [review]
lgtm 🎉

**Review by mhstrkmp — CHANGES_REQUESTED** [review]
- Something seemes broken
- It always loads the same question any idea @AlexisRoe

File: src/App.js [hunk 1]
```diff
@@ -74,26 +67,22 @@ function App() {
...
     }
 
-    gameInfo = await getQuestionInfo(playerName);
-    points.innerText = gameInfo.score;
+    gameInfo = await nextGameInfo;
+    points.innerText = getScore(playerName);
     setTimeout(() => {
       generateGameField();
     }, 2500);
   }
```
Comment (AlexisRoe) [inline src/App.js]: we forgot one line of code
```suggestion
    gameInfo = await nextGameInfo;
    points.innerText = getScore(playerName);
    nextGameInfo = getQuestionInfo(playerName);
    setTimeout(() => {
      generateGameField();
    }, 2500);
  }
```

**Review by AlexisRoe — COMMENTED** [review]
sorry, forgot to push the button 🤦‍♂️

**Pull request merged**